app/models/manageiq/providers/ibm_cloud/inventory/parser/vpc.rb

```diff
@@ -200,14 +217,25 @@ def floating_ips
 
   def volumes
     collector.volumes.each do |vol|
+      az_name = vol&.dig(:zone, :name)
+      attachments = vol&.dig(:volume_attachments)
+      bootable = attachments.any?
+      if bootable
+        attachments.each do |vol_attach|
+          bootable = vol_attach[:type] == "boot"
+          break if bootable
```

Comment:
This looks like it can be `bootable_volume = attachments.to_a.detect { |vol| vol[:type] == "boot" }`

Reply:
nice - much cleaner - will switch

Reply:
fixed